bait-files/src/main/webapp/review.js

```diff
@@ -4,11 +4,45 @@ function getPanels() {
     fold.addEventListener('click', function() {
       this.classList.toggle('active');
       const panel = this.nextElementSibling;
-      if (panel.style.maxHeight) {          //when accordion button is clicked, if panel is visible closes panel by setting maxHeight to null
+      if (panel.style.maxHeight) {  // when accordion button is clicked, if
+                                    // panel is visible closes panel by setting
+                                    // maxHeight to null
         panel.style.maxHeight = null;
       } else {
-        panel.style.maxHeight = panel.scrollHeight + 'px';  //when accordion button is clicked, if panel is not visible, set maxHeight so panel is displayed
+        panel.style.maxHeight = panel.scrollHeight +
+            'px';  // when accordion button is clicked, if panel is not visible,
+                   // set maxHeight so panel is displayed
       }
     });
   }
 }
+
+function getReview() {
+  const queryString = window.location.search;
+  fetch('/review' + queryString)
```

Comment:
Similar to my other comment about Request vs Review, if this method is retrieving a Request object, it should probably be called getRequest and call the '/request' endpoint.  And then I'm guessing the plan is for after the agent has made their determinations and goes to submit the review, you can then package it all into a Review object and call POST on '/review'?